spec/tictactoe_spec.rb

```diff
@@ -8,73 +8,57 @@ class FakeIO
   def initialize(stdin: '', stdout: [])
     @stdin = stdin
     @stdout = stdout
+    @accepted_input = [*1..9].map!(&:to_s)
   end
 
-  def puts(string)
+  def output_to_user(string)
     @stdout << string
   end
 
-  def gets
-    string_to_return = @stdin
+  def prompt_user_for_position
+    string_to_return = @accepted_input.include?(@stdin) ? @stdin.to_i - 1: nil
     @stdin = ''
     string_to_return
   end
 end
 
+
```

Comment:
Some extra whitespace?

Reply:
If you haven't been introduced to the class of tools known as linters, I want to take the opportunity to mention them. I like them a lot! They can differ quite a bit across languages, but the general thing they help you with is picking up code patterns/formatting that you want to stick to, and pointing out when you aren't sticking with them. Often the tool can automatically fix formatting mistakes, and sometimes change your code for the better too!

It's been a while since I've worked in ruby, but the de-facto linting tool may be Rubocop: https://rubocop.readthedocs.io/en/stable/